website/client/src/components/groups/group.vue

```diff
@@ -225,12 +230,15 @@
     box-shadow: 0 2px 2px 0 rgba(26, 24, 29, 0.16), 0 1px 4px 0 rgba(26, 24, 29, 0.12);
     padding: 1em;
     text-align: center;
-    min-width: 80px;
-    max-width: 120px;
+    min-width: 100px;
```

Comment:
I'm not sure exactly why give the min-width was set to 80px but it looks like the item here is currently 120px while with the PR it shrank a bit. Could we change it so that by defaults it's 120px wide?

Reply:
Fixed!